src/Sylius/Component/Translation/Factory/TranslatableFactoryInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Translation\Factory;
+
+use Sylius\Component\Resource\Factory\FactoryInterface;
+
+/**
+ * Interface for factories creating translatable resources.
+ *
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+interface TranslatableFactoryInterface extends FactoryInterface
```

Comment:
What's the point with `TranslatableFactoryInterface`? IMHO it would have the same methods as `FactoryInterface`, so it will be empty, wouldn't it? If we only overwrite the phpdocs with `@return AbstractTranslatable` (not sure about the classname though :)), it will have more sense.